Calculator/Calculator/Model/StringExtension.swift

```diff
@@ -1,8 +1,38 @@
 //  Calculator - StringExtension.swift
 //  created by vetto on 2023/01/27
 
+import Foundation
+
 extension String {
     func split(with target: Character) -> [String] {
         return self.split(separator: target).map { String($0) }
     }
+    
+    func numberFormatting() -> String {
+        var stringArray = self.split(with: ".")
+        stringArray[0] = integerFormatting(stringArray[0])
+        if self.contains(".") {
+            if Double(stringArray[1]) == 0 {
+                stringArray.removeLast()
+            }
+        }
+        
+        return stringArray.joined(separator: ".")
+    }
+    
+    func integerFormatting(_ input: String) -> String {
+        var numberFormatter: NumberFormatter {
+            let numberFormatter = NumberFormatter()
+            numberFormatter.numberStyle = .decimal
+            numberFormatter.maximumFractionDigits = 20
+            numberFormatter.roundingMode = .halfUp
+            
+            return numberFormatter
+        }
```

Comment:
```suggestion
            let numberFormatter = NumberFormatter()
            numberFormatter.numberStyle = .decimal
            numberFormatter.maximumFractionDigits = 20
            numberFormatter.roundingMode = .halfUp
```

이렇게 바로 사용하는게 더 간편하지 않을까요 🤔

Reply:
리뷰어 조언대로 바로 사용할 수 있게 바꾸었습니다.  15445047f31b46b64deaf1771893ac013c8fac58